dashi/bcl2fastq/index_summary.py

```diff
@@ -141,11 +143,11 @@ def create_unknown_index_bar(pruned):
               """
 
     data_unknown = []
-    for lane, d in pruned.groupby("LaneNumber"):
+    for lane, d in pruned.groupby(un_col.LaneNumber):
         data_unknown.append(
             {
                 "x": list(d["index"]),
```

Comment:
Why is this one not usinga  constant?

Reply:
I have made columns defined within a module module-level constants.